odoo/addons/base/models/ir_qweb.py

```diff
@@ -844,7 +896,19 @@ def _prepare_globals(self):
             'Mapping': Mapping,
             'Markup': Markup,
             'escape': escape,
+            'is_string': lambda item: isinstance(item, str),
+            'is_error_to_pass_up': lambda error: isinstance(error, TransactionRollbackError), # for SerializationFailure in assets
+            'is_allowed_cached_value': lambda item: isinstance(item, (str, int, float, bool)),
```

Comment:
those can be changed into inline code, they only use the safe `isinstance` built-in and some safe types